packages/cascara/src/modules/DataUrl/DataUrl.js

```diff
@@ -54,6 +54,7 @@ const DataUrl = ({
     <span>
       {label && isLabeled && <span className={styles.LabelText}>{label}</span>}
       <span aria-label={label} className={styles.Url} {...rest}>
+        {' '}
```

Comment:
This is not needed. It is also why the snapshot changed.
```suggestion
```

Reply:
I am seeing the comment in the PR description stating that the reason for the Snapshot change is `Updated since some components had incorrect names.` but there was no name change that I can see in this PR. Was there something you saw elsewhere, maybe in Github that made you think this was the reason for the Snapshot change?

Reply:
@brianespinosa No, we had to update the fixtures to output the correct component names, like this [one](https://github.com/Espressive/cascara/pull/316/files#diff-8829e36db7e52ce8cec9918fe6748f20ddb43e1e4f1470459bad1d08b2c49aa4). As well as fixing other minor issues in fixtures.

Reply:
The reason why the snapshots changed is that tests use fixtures for testing. i.e. Changing a fixture triggers a change in the snapshot tests.